### ISSUE TITLE
App possibly ripping off Unit Converter Ultimate

### ISSUE THREAD

**Issue by lokesh-krishna** [physphil/UnitConverterUltimate#109]
Found this app on the Play Store: [Metric Conversion](https://play.google.com/store/apps/details?id=com.metric.unitconverter.measurementconverter).

To my eyes, it seems like the app is just a rebranded Unit Converter Ultimate with ads slapped on it.

Is there anything that can be done about this?

**Comment by physphil** [comment]
Yeah, unfortunately I think you're right.  The app is open source and unfortunately there are a lot of clones out there who have just taken the code and put ads in it :(  It's very frustrating.  I can certainly contact the developer and/or Google but I don't actually know if it anything can be done.  I have a list of clones right now and I'll add this to it.  Thank you for bringing it to my attention!

**Comment by tacsipacsi** [comment]
I think it violates the license conditions by not attributing you as an author and not making the source code available to users. I don’t know the Google Play Terms of Service, it may violate them as well by publishing a clone of an app already available there.

**Comment by avipars** [comment]
This one also: https://play.google.com/store/apps/details?id=com.digitalindeed.converter

Try issuing a DMCA Takedown Request.
